package.json

```diff
@@ -20,5 +20,6 @@
     "install-and-build": "node ./scripts/install-and-build",
     "deploy": "node ./scripts/deploy",
     "start": "node ./scripts/run-frontend"
-  }
+  },
+  "dependencies": {}
```

Comment:
It got added when I accidentally added some dependencies to the root repo instead of the frontend repo. I don't see any reason to remove it necessarily, but I can.